app/src/main/res/xml/about.xml

```diff
@@ -160,5 +173,12 @@
         <me.jfenn.attribouter.wedges.TranslatorWedge
             name="Demetrio F. B."
             locales="ro,de" />
+
+        <me.jfenn.attribouter.wedges.TranslatorWedge
+            name="@string/contributor_german_name"
+            avatar="https://avatars2.githubusercontent.com/u/16625266"
+            locales="ru"
```

Comment:
`lcalces="ru"` doesn't work, but mb in next updated will be supported.

Reply:
It needs to be patched to support language + country codes:

https://github.com/fennifith/Attribouter/blob/bb4361ed267f787d89b18733052be9c736cd4a43/attribouter/src/main/java/me/jfenn/attribouter/wedges/TranslatorWedge.kt#L85-L90

```
>>> Locale("ru", "RU").getDisplayCountry()
res30: kotlin.String! = Russia
>>> Locale("ru").getDisplayCountry()
res31: kotlin.String! = 
```